<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Login</class>
  <widget class="QWidget" name="Login">
    <property name="windowModality">
      <enum>Qt::NonModal</enum>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>649</width>
        <height>431</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>449</width>
        <height>271</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>铁路隧道建筑限界测量系统</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../resource/icons.qrc">
        <normaloff>:/image/icon.png</normaloff>:/image/icon.png
      </iconset>
    </property>
    <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
        <rect>
          <x>-10</x>
          <y>0</y>
          <width>661</width>
          <height>431</height>
        </rect>
      </property>
      <property name="minimumSize">
        <size>
          <width>451</width>
          <height>271</height>
        </size>
      </property>
      <property name="styleSheet">
        <string notr="true">border-image: url(:/image/background.png);</string>
      </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
        <rect>
          <x>190</x>
          <y>320</y>
          <width>249</width>
          <height>105</height>
        </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
          <widget class="QLabel" name="name">
            <property name="minimumSize">
              <size>
                <width>81</width>
                <height>20</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>81</width>
                <height>20</height>
              </size>
            </property>
            <property name="font">
              <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
              </font>
            </property>
            <property name="styleSheet">
              <string notr="true"/>
            </property>
            <property name="text">
              <string>用户名</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignCenter</set>
            </property>
          </widget>
        </item>
        <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="nameedit">
            <property name="minimumSize">
              <size>
                <width>133</width>
                <height>20</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>133</width>
                <height>20</height>
              </size>
            </property>
            <property name="font">
              <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
              </font>
            </property>
          </widget>
        </item>
        <item row="1" column="0">
          <widget class="QLabel" name="password">
            <property name="minimumSize">
              <size>
                <width>81</width>
                <height>20</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>81</width>
                <height>20</height>
              </size>
            </property>
            <property name="font">
              <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
              </font>
            </property>
            <property name="styleSheet">
              <string notr="true"/>
            </property>
            <property name="text">
              <string>密码</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignCenter</set>
            </property>
          </widget>
        </item>
        <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="passwordedit">
            <property name="minimumSize">
              <size>
                <width>133</width>
                <height>20</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>133</width>
                <height>20</height>
              </size>
            </property>
            <property name="font">
              <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
              </font>
            </property>
            <property name="echoMode">
              <enum>QLineEdit::Password</enum>
            </property>
          </widget>
        </item>
        <item row="2" column="0" colspan="3">
          <widget class="QLabel" name="label">
            <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="alignment">
              <set>Qt::AlignCenter</set>
            </property>
          </widget>
        </item>
        <item row="3" column="0">
          <widget class="QPushButton" name="loginButton">
            <property name="enabled">
              <bool>false</bool>
            </property>
            <property name="font">
              <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
              </font>
            </property>
            <property name="text">
              <string>登录(&amp;L)</string>
            </property>
            <property name="icon">
              <iconset resource="../../resource/icons.qrc">
                <normaloff>:/image/user.png</normaloff>:/image/user.png
              </iconset>
            </property>
            <property name="default">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item row="3" column="1">
          <widget class="QPushButton" name="exitButton">
            <property name="font">
              <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
              </font>
            </property>
            <property name="text">
              <string>退出(&amp;Q)</string>
            </property>
            <property name="icon">
              <iconset resource="../../resource/icons.qrc">
                <normaloff>:/image/cross.png</normaloff>:/image/cross.png
              </iconset>
            </property>
            <property name="default">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item row="3" column="2">
          <widget class="QPushButton" name="helpButton">
            <property name="font">
              <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
              </font>
            </property>
            <property name="text">
              <string>关于(&amp;A)</string>
            </property>
            <property name="icon">
              <iconset resource="../../resource/icons.qrc">
                <normaloff>:/image/alert.png</normaloff>:/image/alert.png
              </iconset>
            </property>
            <property name="default">
              <bool>true</bool>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QWidget" name="">
      <property name="geometry">
        <rect>
          <x>170</x>
          <y>210</y>
          <width>291</width>
          <height>108</height>
        </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
          <spacer name="horizontalSpacer_3">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>40</width>
                <height>20</height>
              </size>
            </property>
          </spacer>
        </item>
        <item row="0" column="1">
          <spacer name="horizontalSpacer">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>40</width>
                <height>20</height>
              </size>
            </property>
          </spacer>
        </item>
        <item row="0" column="2">
          <widget class="QRadioButton" name="userButton_2">
            <property name="minimumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="iconSize">
              <size>
                <width>40</width>
                <height>40</height>
              </size>
            </property>
          </widget>
        </item>
        <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>40</width>
                <height>20</height>
              </size>
            </property>
          </spacer>
        </item>
        <item row="1" column="0">
          <widget class="QRadioButton" name="userButton_4">
            <property name="minimumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="iconSize">
              <size>
                <width>40</width>
                <height>40</height>
              </size>
            </property>
          </widget>
        </item>
        <item row="1" column="3">
          <spacer name="horizontalSpacer_4">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>40</width>
                <height>20</height>
              </size>
            </property>
          </spacer>
        </item>
        <item row="0" column="0">
          <widget class="QRadioButton" name="userButton_1">
            <property name="minimumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="iconSize">
              <size>
                <width>40</width>
                <height>40</height>
              </size>
            </property>
          </widget>
        </item>
        <item row="1" column="2">
          <widget class="QRadioButton" name="userButton_5">
            <property name="minimumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="iconSize">
              <size>
                <width>40</width>
                <height>40</height>
              </size>
            </property>
          </widget>
        </item>
        <item row="1" column="4">
          <widget class="QRadioButton" name="userButton_0">
            <property name="minimumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="iconSize">
              <size>
                <width>40</width>
                <height>40</height>
              </size>
            </property>
          </widget>
        </item>
        <item row="0" column="4">
          <widget class="QRadioButton" name="userButton_3">
            <property name="minimumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>80</width>
                <height>40</height>
              </size>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="iconSize">
              <size>
                <width>40</width>
                <height>40</height>
              </size>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <tabstops>
    <tabstop>nameedit</tabstop>
    <tabstop>passwordedit</tabstop>
    <tabstop>loginButton</tabstop>
    <tabstop>helpButton</tabstop>
    <tabstop>exitButton</tabstop>
  </tabstops>
  <resources>
    <include location="../../resource/icons.qrc"/>
  </resources>
  <connections>
    <connection>
      <sender>exitButton</sender>
      <signal>clicked()</signal>
      <receiver>Login</receiver>
      <slot>close()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>241</x>
          <y>232</y>
        </hint>
        <hint type="destinationlabel">
          <x>121</x>
          <y>201</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
